Podcasts.md

```diff
@@ -38,6 +38,7 @@
 - [Self-Taught or Not](https://www.selftaughtornot.com/): A podcast covering the do's and don'ts of software development, hosted by two software developers; one self-taught, and one not. Topics include advice for developers, technologies, interviews, and more.
 - [Ladybug Podcast](https://www.ladybug.dev/): This podcast co-hosted by 3 ladies who wanted to add their voices to the space and share their experiences and advice. They have great discussions around how to start coding, the hot technologies right now, how to get your first developer job, and more!
 - [The Stack Overflow Podcast](https://stackoverflow.blog/podcast/): The Stack Overflow podcast is a weekly conversation about working in software development, learning to code, and the art and culture of computer programming. Hosted by Sara Chipps, Paul Ford, and Ben Popper, the series will feature questions from our community, interviews with fascinating guests, and hot takes on what’s happening in tech.
+- [HTML All The Things](https://www.htmlallthethings.com/podcast): Join hosts Matt Lawrence and Mike Karan as they dive into the world of web development and design. In this show, they discuss running a small business and making websites and web apps for customers from all over the industry.  Matt and Mike also take a deep dive into things like their latest projects, their business strategies, and much more! 
```

Comment:
Regarding the following:  
`they discuss running a small business and making websites and web apps for customers from all over the industry.`

Could be improved with the following - avoiding repetitive use of  "and"
`they discuss running a small business, making websites and web apps for customers from all over the industry.`

Reply:
Yeah that sounds better.